### PR TITLE
[IMP] show knowledge and social module in apps menu in community

### PR DESCRIPTION
**Pull request by niyasraphy**
**Description of the issue/feature this PR addresses:** 
Display knowledge and social module in odoo community similar to other enterprise modules

**Current behavior before PR:** 
Knowledge and social module is not shown as it is not added in data

**Desired behavior after PR is merged:** 
Knowledge and social module will be listed in community with upgrade button

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by niyasraphy** [comment]
@mart-e can you review this if you have some time ?

**Comment by niyasraphy** [comment]
@abd-msyukyu-odoo hi, if you have some time, can you review this pr

File: odoo/addons/base/data/ir_module_module.xml [hunk 1]
```diff
@@ -44,6 +44,20 @@
             <field name="website">https://www.odoo.com/app/accounting?utm_source=db&amp;utm_medium=module</field>
         </record>
 
+        <record model="ir.module.module" id="base.module_knowledge">
+            <field name="name">knowledge</field>
+            <field name="sequence">30</field>
+            <field name="shortdesc">Knowledge</field>
+            <field name="category_id" ref="base.module_category_productivity_knowledge"/>
```
Comment (abd-msyukyu-odoo) [inline odoo/addons/base/data/ir_module_module.xml:51]: Could you use `"base.module_category_productivity"` instead (and remove the `module_category_productivity_knowledge` record) ?

**Review by abd-msyukyu-odoo — CHANGES_REQUESTED** [review]
Hello @niyasraphy , thank you for your feedback ! I only have a small comment regarding the category.

Also, if you want to (no pressure, I'll do it myself if you don't :smile: ), you could add a similar fix for `Social Marketing` (`social` module) using the existing `base.module_category_marketing` category, 
and update the icon and category for `Appointments` (`appointment` module) to `base.module_category_marketing`

Cheers!

File: odoo/addons/base/data/ir_module_module.xml (hunk 1 above)
Comment (niyasraphy) [reply odoo/addons/base/data/ir_module_module.xml]: hi @abd-msyukyu-odoo , thanks for the feedback i have updated the category as suggested. but in the knowledge module manifest file the category is given as Productivity/Knowledge, so should we need to change here also ?

looking for your thoughts

**Comment by niyasraphy** [comment]
more than happy to do the suggested changes for social and appointment modules, and I always welcome it.
I have created a separate pr: https://github.com/odoo/enterprise/pull/33522

Regarding the change with icon of Appointments module, i should replace with which icon ?
Thanks in advance

File: odoo/addons/base/data/ir_module_module.xml (hunk 1 above)
Comment (abd-msyukyu-odoo) [reply odoo/addons/base/data/ir_module_module.xml]: Hello again! No need to change the Enterprise category for Knowledge.

Indeed, in Enterprise the category is different, and its purpose is to contain future optional modules for Knowledge. But in Odoo Community, the `ir.module.module` record only redirects to the Odoo website (to propose the upgrade plans, or to more information about the module), so we don't need the specific category. That record is not used when using Odoo Enterprise.

Comment (niyasraphy) [reply odoo/addons/base/data/ir_module_module.xml]: @abd-msyukyu-odoo thanks for the clarification :)

**Comment by niyasraphy** [comment]
hi, should i do the rebasing as one of the test case is failed ?

**Comment by abd-msyukyu-odoo** [comment]
@niyasraphy I'm sorry for the confusion, I meant that you could add/change `ir.module.module` records for social and appointment for Odoo community (to also propose the upgrade plan), not for Enterprise (those categories are corrects)

**Comment by abd-msyukyu-odoo** [comment]
> hi, should i do the rebasing as one of the test case is failed ?

Yes you should try rebasing, the error seems unrelated to your commit

**Comment by abd-msyukyu-odoo** [comment]
@niyasraphy I'll provide you with the icon later as I'm on my phone right now and it is not convenient 😅

**Comment by abd-msyukyu-odoo** [comment]
> @abd-msyukyu-odoo i closed the other PR i created ;) Thanks for the clarifying it for me and i think this time i updated it correctly, updated appointment icon with the icon in the enterprise for the appointments module and category for the appointment is also updated to base.module_category_productivity.
> 
> But i couldn't find social marketing module in community (in data files), does i miss something ? can you confirm this, if it is not there, should we consider adding it too ?
> 
> Thanks for the time.

Yes indeed the data for social marketing is missing and we should add it (like knowledge)

Also, the category for both appointment and social should be `module_category_marketing` (not productivity)

You too, thanks for your time!

**Comment by niyasraphy** [comment]
@abd-msyukyu-odoo updated as per the suggestion ad modified the commit message and pr title

**Review by abd-msyukyu-odoo — APPROVED** [review]

**Comment by niyasraphy** [comment]
@abd-msyukyu-odoo hi, is there a way to ping or notify the pending reviewers
thanks

**Comment by abd-msyukyu-odoo** [comment]
> @abd-msyukyu-odoo hi, is there a way to ping or notify the pending reviewers thanks

Hello @niyasraphy Unfortunately both our functional and technical reviewers are on holidays right now, but don't worry, there is a task for this PR internally and it will most likely be merged next week when they come back :)

**Comment by niyasraphy** [comment]
thanks @abd-msyukyu-odoo

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**